04/app.js

```diff
@@ -5,21 +5,18 @@ function onLoadedCont() {
     element.addEventListener('click', addClass)
   });
 
-  // setTimeout(() => {
-  //   this.classList.add('clicked');
-  //   this.stopPropagation()
-  // }, 300)
-
-  function addClass(e) {
+  function addClass() {
     setTimeout(() => {
       this.classList.add('clicked');
     }, this.dataset.time)
   }
 
-  function removeClass() {
-    divList.forEach(element => {
+  function removeClass(e) {
+    if (e.target === document.body) {
```

Comment:
Ok, chociaż myślałem też o `e.target === e.currentTarget`.
Wtedy jest to uniwersalne rozwiązanie, na sprawdzenie czy ma tutaj miejsce propagacja czy nie.